src/types.js

```diff
@@ -195,43 +195,146 @@ export type TaskChange = {
   external_id?: string,
 };
 
-/* Stories */
-
-export type StoryType = 'bug' | 'chore' | 'feature';
+/* Identity */
 
-export type Story = {
+export type Identity = {
   entity_type: string,
+  name: string | null,
+  type: string | null,
+};
+
+/* PullRequest */
+
+export type PullRequest = {
+  branch_id: number,
+  closed: boolean,
   created_at: string,
+  entity_type: string,
+  id: number,
+  num_added: number,
+  num_commits: number,
+  num_modified: number,
+  num_removed: number,
+  number: number,
+  target_branch_id: number,
+  title: string,
   updated_at: string,
-  id: ID,
-  external_id: string,
+  url: string,
+};
+
+/* Branch */
+
+export type Branch = {
+  created_at: string | null,
+  deleted: boolean,
+  entity_type: string,
+  id: number | null,
+  merged_branch_ids: Array<number>,
   name: string,
-  story_type: StoryType,
-  description: string,
-  archived: boolean,
+  persistent: boolean,
+  pull_requests: Array<PullRequest>,
+  repository_id: number | null,
+  updated_at: string | null,
+  url: string,
+};
+
+/* Comment */
+
+export type Comment = {
+  author_id: string | null,
+  created_at: string,
+  entity_type: string,
+  external_id: string | null,
+  id: number,
+  mention_ids: Array<string>,
   position: number,
-  workflow_state_id: ID,
-  started: boolean,
-  started_at: string,
-  started_at_override: string,
-  completed: boolean,
-  completed_at: string,
-  completed_at_override: string,
+  story_id: number,
+  text: string,
+  updated_at: string | null,
+};
+
+/* Commit */
+
+export type Commit = {
+  author_email: string,
+  author_id: string | null,
+  author_identity: Identity,
+  created_at: string,
+  entity_type: string,
+  hash: string,
+  id: number | null,
+  merged_branch_ids: Array<number>,
+  message: string,
+  repository_id: number | null,
+  timestamp: string,
+  updated_at: string | null,
+  url: string,
+};
+
+/* StoryLink */
+export type StoryLinkVerb = 'blocks' | 'duplicates' | 'relates to';
+
+export type StoryLink = {
+  id: ID,
+  created_at: string,
+  updated_at: string,
+  type: string,
+  subject_id: ID,
+  object_id: ID,
+  verb: StoryLinkVerb,
+};
+
+export type StoryLinkChange = {
+  object_id: ID,
+  verb_id: ID,
+  verb: StoryLinkVerb,
+};
+
+/* Stories */
+
+export type StoryType = 'bug' | 'chore' | 'feature';
+
+export type StorySearch = {
```

Comment:
I don't quite love having two different types for `Story` here, but we definitely do return two types from our API.

For consistency sake, can you call this `StorySlim` (which is what this is called in our REST API codebase)?

Reply:
Have a look at the https://clubhouse.io/api/rest/beta/#StorySearchResults. If the API returns a `StorySlim`, the API documentation for the type of the field `data` needs an update.

Note: it is the same in the documentation of the V2: https://clubhouse.io/api/rest/v2/#StorySearchResults